packages/web/src/scss/components/Stack/_Stack.scss

```diff
@@ -9,7 +10,22 @@
 }
 
 .Stack--hasSpacing {
-    gap: theme.$gap;
+    $property-name: --stack-spacing;
+    $fallback: theme.$gap;
```

Comment:
How about nested `Stack`s? Or is it in the scope of the `StackItem` issue?

Reply:
Ha, true, I will test if it doesn't break.